bin/workspace_status.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash
```

Comment:
As mentioned in the `.bazelrc` file comment, this file is not needed anymore due to rules_docker improvements.